### ISSUE TITLE
Is there a working codec that can be installed on Win10 for viewing files?

### ISSUE THREAD

**Issue by stewmakesmovies** [gopro/cineform-sdk#28]
After a couple years, i'm trying to view some old HD footage with cineform compression, but don't have the codec.  Is this codec available any more so I can simply view/transcode my old cineform avi files?  I can't get past the no-longer-supported license key step when trying to install the old version of neo3d I have archived.  Any help would be appreciated, thanks!

**Comment by dnewman-gpsw** [comment]
Yes, the developer was an early supporter of this open source effort.
